en/components/linear_progress.md

```diff
@@ -9,29 +9,267 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 <div class="divider"></div>
 
 ### Linear Progress Demo
-<div class="sample-container loading" style="height:650px">
-    <iframe id="progressbar-sample-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/progressbar" onload="onSampleIframeContentLoaded(this);"></iframe>
+<div class="sample-container loading" style="height:550px">
+    <iframe id="progressbar-sample-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/linear-progressbar-sample-1" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
 <div>
 <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="progressbar-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 <div class="divider--half"></div>
 
 ### Usage
+To get started with the Ignite UI for Angular Linear Progress, we should first import the **IgxProgressBarModule** in the **app.module.ts** file:
+```typescript
+// app.module.ts
+
+...
+import { IgxProgressBarModule } from 'igniteui-angular/main';
+
+@NgModule({
+    ...
+    imports: [..., IgxProgressBarModule],
+    ...
+})
+export class AppModule {}
+```
+
+And now let's create a simple example in which we are going to simulate a continuous process, that is triggered on button click. In addition you can notice
+that the speed of loading depends on the `max` attribute that we are set to our `igx-linear-bar`. In first case our max value is set to 200 and loads quite slower,
```

Comment:
that we have set to our `igx-linear-bar` 
or
that we are setting to our `igx-linear-bar`.
In *the* first case, our max value is set to 200 and loads slower than in the second where *it* is set to 100